lib/reline/line_editor.rb

```diff
@@ -3345,4 +3347,7 @@ def finish
     @mark_pointer = new_pointer
   end
   alias_method :exchange_point_and_mark, :em_exchange_mark
+
+  private def em_meta_next(key)
+  end
```

Comment:
In method `process_key`, `ed_insert` is called if method_symbol is not defined.
I think adding this method is the simplest and easiest way to prevent ESC insertion.
Although, not all method_symbol written in `lib/reline/key_actor/emacs.rb` is defined in `line_aditor.rb` and I think it does not causing problem. Perhaps there is another way to fix without defining this method.